### ISSUE TITLE
[Feature] add statistics related to feed configuration to make configuration and import/export easier on not managed instances

### ISSUE THREAD

**Issue by geogiraux** [FreshRSS/FreshRSS#6708]
**Is your feature request related to a problem? Please describe.**
I had to switch from one hosted FreshRSS instance to another one
Issue [#5914](https://github.com/FreshRSS/FreshRSS/issues/5914) is quite related

**Describe the solution you’d like**
Adding a specific stat page (since it's not really linked to feed contents) with :
- number of feeds (enabled and disabled)
- distribution of feeds depending on refreshing interval
- number of feeds using a proxy
- number of feeds bypassing safety certificate verification

**Describe alternatives you’ve considered**
export opml file and manipulate it with filtering and counting lines functions (for the included parameters like proxy) 

**Additional context**
nothing comes to my mind but feel free to ask ;)

**Comment by math-GH** [comment]
it is a very very edgy case. I do not see a reason to have it implemented in the core.
If really needed maybe an extension could be the solution.

**Comment by geogiraux** [comment]
I quite agree with you :) even if I had also in mind admin system fighting with performances issues to quickly visualize the settings done by their users. another parameter I forgot to mention is the wait delay...
Maybe just add to the core the number of feeds on the statistics page ?

**Comment by Alkarex** [comment]
> Maybe just add to the core the number of feeds

Like in the *Manage users* page?

![image](https://github.com/user-attachments/assets/ac969baa-b9a9-4bb2-866a-8f2b25e50cb4)

**Comment by Alkarex** [comment]
> Maybe just add to the core the number of feeds on the statistics page

Like in the *Manage users* page?

![image](https://github.com/user-attachments/assets/f50f1f90-3df8-4d60-9463-8e5f894086f8)

**Comment by Alkarex** [comment]
By the way, some statistics are also available by CLI https://github.com/FreshRSS/FreshRSS/blob/edge/cli/README.md

See in particular `./cli/user-info.php`

**Comment by geogiraux** [comment]
> Like in the Manage users page?

Indeed... as in the case mentioned, I don't manage my instance so I wasn't aware of this page dedicated to admin, neither the CLI
